developer/tutorial/plugin-layouts-3.md

```diff
@@ -69,12 +69,13 @@ We also need to add our layout to the registry via our `register.js`. We are goi
 ```js
 layout: [{
   layout: "coreLayoutBeesknees",
-  workflow: "coreProductWorkflow",
+  workflow: "coreWorkflow",
```

Comment:
I think to keep things focused, I'll split this into two:
1.  This PR will focus on updating Reaction docs based on the work I did on [custom layouts](https://github.com/reactioncommerce/reaction/pull/2023)

2. Another ticket/issue to focus on fixing the inconsistency I noticed in the Layout tutorial (which I discovered while working on *1 above).
Your comment above falls under this second part. So, I'll recreate and test based on your comment separately for *2